src/servers/graphql-server.ts

```diff
@@ -66,21 +68,8 @@ export const isEditor = rule({ cache: "contextual" })((parent, args, ctx) => {
 const geeTestConfig = getGeetestConfig()
 const geetest = Geetest(geeTestConfig)
 
-const sessionContext = ({
-  token,
-  ips,
-  body,
-  apiKey,
-  apiSecret,
-}): Promise<GraphQLContext> => {
```

Comment:
Why remove the `Promise<GraphQLContext>` type here?

Reply:
@nicolasburtey was there a reason you removed this?

Reply:
probably no good reasons

Reply:
I've added it back in.